analyzers/tests/SonarAnalyzer.Test/TestCases/AspNet/AvoidUnderPosting.cs

```diff
@@ -17,8 +18,10 @@ public class ModelUsedInController
 //                 ^^^^^^^^^^^^^
         public int? NullableValueProperty { get; set; }
         [Required] public int RequiredValueProperty { get; set; }                   // Noncompliant, RequiredAttribute has no effect on value types
-        [Range(0, 10)] public int ValuePropertyWithRangeValidation { get; set; }    // Noncompliant
+        [Range(0, 10)] public int ValuePropertyWithRangeValidation { get; set; }    // Compliant
         [Required] public int? RequiredNullableValueProperty { get; set; }
+        [JsonProperty(Required = Required.Always)] public int JsonRequiredValuePropertyAlways { get; set; }         // Compliant
```

Comment:
Add test cases for [Default and DisallowNull ](https://www.newtonsoft.com/json/help/html/t_newtonsoft_json_required.htm).